### ISSUE TITLE
why not make image_size as a variable instead of a constant?

### ISSUE THREAD

**Issue by tenbabagu** [titu1994/Inception-v4#6]
why not make image_size as a variable instead of a constant?

**Comment by titu1994** [comment]
You're right. This was very early code when it came out. It didn't follow Keras standards to model building. 

Since Keras now comes with InceptionV4 built into applications, this can be considered more or less deprecated.
